dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:13-alpine
```

Remember always use lts node version

```suggestion
FROM node:lts-alpine3.11
```